src/components/SuspiciousCard.tsx

```diff
@@ -0,0 +1,122 @@
+import i18n from 'assets/i18n';
+import React, { useEffect, useState } from 'react';
+import { View, StyleSheet, Dimensions } from 'react-native';
+import { Text } from 'react-native-paper';
+import { ipctColors } from 'styles/index';
+
+import NoSuspiciusBadgeOutlineSvg from './svg/NoSuspiciusBadgeOutlineSvg';
+import WarningTriangle from './svg/WarningTriangle';
+
+export default function SuspiciousCard(props: { suspectCounts: number }) {
+    const { width } = Dimensions.get('screen');
+    const [cardText, setCardText] = useState<string>();
+    const [cardLogo, setCardLogo] = useState<JSX.Element>();
+
+    useEffect(() => {
+        const text = setTextSuspects(props.suspectCounts);
+        const logo = setLogoSuspects(props.suspectCounts);
+
+        setCardLogo(logo);
+        setCardText(text);
+    }, [props.suspectCounts]);
+
+    const setLogoSuspects = (suspects: number) => {
+        switch (true) {
+            case suspects < 1:
+                return (
+                    <NoSuspiciusBadgeOutlineSvg
+                        style={{ marginRight: 12, marginTop: 4 }}
+                    />
+                );
+
+            case suspects < 4:
+                return (
+                    <WarningTriangle
+                        color="#73839D"
```

Comment:
add it to `ipctColors` if not there yet and use here.

Reply:
Fixed.